### DISCUSSION TITLE
Connector for Safesoft FME

### DISCUSSION THREAD

**Discussion by vdl-vciriza** [open-metadata/OpenMetadata#9625] [Ideas]
Teams working on geospatial information have Safesoft FME among their ETL toolset in many cases. 

The possibility of connecting to the FME servers APIs to audit and track the different datasources and pipelines would be great as it would give us an excellent visibility on the data sources, users and processes.

FME provides a REST API to plug in a connector:
https://docs.safe.com/fme/html/FME_REST/apidoc/v3/index.html

**Comment by pinkerltm** [comment]
Although I like FME Server very much (its a great product!) I doubt that very much teams can afford it for their daily spatial ETL pipelines. Don't get me wrong, I would very much like to see more integration with spatial toolsets also. I have a very similar request [here](https://github.com/open-metadata/OpenMetadata/issues/13009) but we are moving away from FME to GDAL/OGR Jobs scheduled by any other scheduler (Airflow, Prefect or even the bare windows scheduler).

Perhaps a good basic integration would be to just parse .FME Files to get the basic ETL Lineage (I would really need that right away), and then for sure also metadata from the FME Server (last run, etc...) if its available.

Curious where OM will go spatially...
